mono/mini/interp/interp.c

```diff
@@ -4478,16 +4483,22 @@ ves_exec_method_with_context (InterpFrame *frame, ThreadContext *context, unsign
 				--sp;
 			}
 			frame->ip = ip;
+
+			if (frame->ex_handler != NULL && MONO_OFFSET_IN_HANDLER(frame->ex_handler, frame->ip - rtm->code)) {
+				MonoException *exc = frame->ex;
+				frame->ex_handler = NULL;
+				frame->ex = NULL;
+				if (exc && exc->object.vtable->klass == mono_defaults.threadabortexception_class &&
+						frame->imethod->method->wrapper_type != MONO_WRAPPER_RUNTIME_INVOKE)
+					THROW_EX (exc, frame->ip);
+			}
```

Comment:
is that something mini also does? soon, we want to get rid of the exception handling in the interpreter

Reply:
jit also rethrows abort exception at the end of catch.

https://github.com/mono/mono/blob/master/mono/mini/method-to-ir.c#L11477